detekt.yml

```diff
@@ -0,0 +1,70 @@
+build:
+  maxIssues: 10 # Change number of issues allowed as per need
+  excludeCorrectable: false
+  weights:
+    complexity: 2
+    LongParameterList: 1
+
+config:
+  validation: true
+  warningsAsErrors: false
+  checkExhaustiveness: false
+  excludes: ''
+
+processors:
+  active: true
+  exclude:
+    - 'DetektProgressListener'
+
+console-reports:
+  active: true
+  exclude:
+    - 'ProjectStatisticsReport'
+    - 'ComplexityReport'
+
+output-reports:
+  active: true
+
+style:
+  MaxLineLength:
+    active: true
+    maxLineLength: 150
+  WildcardImport:
+    active: true
+  UnusedImports:
+    active: true
+  NewLineAtEndOfFile:
+    active: true
+
+naming:
+  PackageNaming:
+    active: true
+  FunctionNaming:
+    active: false  # Composables often start with uppercase
+  TopLevelPropertyNaming:
+    active: false
+  ObjectPropertyNaming:
+    active: true
+
+complexity:
+  LongMethod:
+    active: false   # Disable, Compose UIs can be long
```

Comment:
I had a look at the [default detekt rules](https://github.com/detekt/detekt/blob/v2.0.0-alpha.1/detekt-core/src/main/resources/default-detekt-config.yml#L330) and there is no mentioning of compose there.

Kindly set the rules to the default ones.